.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+## Changes made
+* [Replace with summaries of general changes made]
+* [Did you modify or delete?]
+* [How do your changes fix the issue?]
+
+Closes #[issue number]
```

Comment:
Should be [Issue Link] I think so GitHub auto links it.

Reply:
Ignore me on this one what you have will auto-link https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword